.gitignore

```diff
@@ -177,3 +177,6 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 plugin_src/service_credentials.json
+
+# Generated release notes file
+docs/release-notes.rst
```

Comment:
Should we add the `unreleased` folder here instead of this file? I thought we would like to track the `release-notes` file